.env

```diff
@@ -0,0 +1 @@
+PYTHONPATH=writer:reader:shared
```

Comment:
This file is special for docker compose. Is it there because of dc or vscode?

Reply:
It's for vscode, does it influence dc as well?

Reply:
https://docs.docker.com/compose/environment-variables/#the-env-file